app/scripts/modules/core/src/managed/ManagedResourceObject.tsx

```diff
@@ -56,6 +57,20 @@ const getResourceRoutingInfo = (
   return null;
 };
 
+const resourceStatusConfig: {
+  [status: string]: Pick<IStatusBubbleProps, 'appearance' | 'iconName'>;
+} = {
+  ACTUATING: { appearance: 'progress', iconName: 'mdActuating' },
+  CREATED: { appearance: 'info', iconName: 'mdCreated' },
+  DIFF: { appearance: 'info', iconName: 'mdDiff' },
+  CURRENTLY_UNRESOLVABLE: { appearance: 'error', iconName: 'mdError' },
+  ERROR: { appearance: 'error', iconName: 'mdError' },
+  PAUSED: { appearance: 'neutral', iconName: 'mdPaused' },
```

Comment:
we currently show `PAUSED` as a `warning` state on the Infrastructure views, can we switch to that? While pausing isn't an error state, in terms of the moments where it appears to users the expectation is that in most cases it's something that is non-normal in a probably-not-great way. For example, there's something wrong so you had to pause to fix it, you're not sure what's going on so you paused to figure it out, etc. Even in cases where the reason wasn't necessarily 'bad', it's still important for anyone using the application to be aware that management is paused so they know what to expect.

```suggestion
  PAUSED: { appearance: 'warning', iconName: 'mdPaused' },
```

Reply:
Thanks for pointing this out. I didn't realize that we were already doing this in infrastructure view. The explanation makes a lot of sense.